rust/private/rust.bzl

```diff
@@ -526,16 +504,16 @@ def _rust_library_group_impl(ctx):
     runfiles = []
 
     for dep in ctx.attr.deps:
-        if rust_common.crate_info in dep:
+        if CrateInfo in dep:
             dep_variant_infos.append(rust_common.dep_variant_info(
-                crate_info = dep[rust_common.crate_info] if rust_common.crate_info in dep else None,
-                dep_info = dep[rust_common.dep_info] if rust_common.crate_info in dep else None,
+                crate_info = dep[CrateInfo] if CrateInfo in dep else None,
+                dep_info = dep[DepInfo] if DepInfo in dep else None,
```

Comment:
cleaned up provider references while I was here and discovered `dep_info` was cheking for the CrateInfo provider but using DepInfo one :)